superset/config.py

```diff
@@ -153,6 +153,9 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
 SAMPLES_ROW_LIMIT = 1000
 # max rows retrieved by filter select auto complete
 FILTER_SELECT_ROW_LIMIT = 10000
+# default time filter in explore
+# values may be "Last day", "Last week", "<ISO date> : now", etc.
+DEFAULT_TIME_FILTER = None
```

Comment:
one nite, we should keep this config as before, the `None` and the `No filter` will generate different time expressions. the `None` will generate ` : today`(col <= today()), and `No filter` will generate ` : `(empty filter).

Reply:
Fixed 👍